perfmetrics/scripts/read_cache/run_load_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# echo on
+set -x
+
+set -e
+
+if [[ -z "${WORKING_DIR}" ]]; then
+  echo "Please set the working directory..."
+  exit 1
+fi
+
+num_of_threads=${1:-40}
```

Comment:
can we make num of threads, number of files passable ? Basis on these two we can decide max file size , number of files per thread.

Also, if it takes time, we are fine to not check-in this script for now and check-in once we run initial tests and are sure about the workloads that we will run repeatedly. 

For now, you can just document the below snippet somewhere. 
./mount_gcsfuse.sh -s 950 -c /mnt/disks/local_ssd
workload_dir=$WORKING_DIR/gcs/1K
mkdir -p $workload_dir
number_of_files_per_thread=$((ONE_MILLION / num_of_threads))
./run_read_cache_fio_workload.sh -e 5 -n $number_of_files_per_thread -t $num_of_threads -b 1K -s 1K -d $workload_dir

Reply:
Agreed. Removing this file for now.